index.html

```diff
@@ -0,0 +1,7 @@
+<form method="POST" ACTION="">

```

Comment:
maybe this page and `home.html`/anything not in the `templates` folder should be deleted now? Are they used?